x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/MlSingleNodeTestCase.java

```diff
@@ -46,6 +49,33 @@ protected Settings nodeSettings() {
         return pluginList(LocalStateMachineLearning.class);
     }
 
+    /**
+     * This cleanup is to fix the problem described in
+     * https://github.com/elastic/elasticsearch/issues/38952
+     */
+    @Override
+    public void tearDown() throws Exception {
+        try {
+            logger.info("[{}#{}]: ML-specific after test cleanup", getTestClass().getSimpleName(), getTestName());
```

Comment:
What is the reason for making this info level? I would like to see our test logs have less lines in them, not more.

Reply:
It's info in the same place in `ESSingleNodeTestCase`, so I copied that.  I'm happy to downgrade this one to trace though.

Reply:
Oh, I did not realize that we had an info line there in `ESSingleNodeTestCase` too. Thanks for pointing that out. Yes, I think that not only this one but both can be downgraded to trace. 🙏